### PR TITLE
The great linting exercise of 2020

### PR DESCRIPTION
**Pull request by pckroon**
Due to the failed pylint run on #280 I decided to get some linting done.
Mostly fixing whitespace and indent issues, nothing serious.

### CODE REVIEW

**Comment by pckroon** [comment]
The sudden drop in pylint scores is due to the pylint message "E1102: pytest.mark.parametrize is not callable (not-callable)". This is obviously wrong, and caused by a version increase in pytest. This seems to be fixed in pytest upstream. However, since it would cause loud errors anyway, I've disabled it in the pylintrc

File: vermouth/gmx/itp_read.py [hunk 1]
```diff
@@ -271,7 +271,8 @@ def _interactions(self, line, lineno=0):
             atom_idxs=atom_idxs,
             )
 
-    def _split_atoms_and_parameters(self, tokens, atom_idxs):
+    @staticmethod
+    def _split_atoms_and_parameters(tokens, atom_idxs):
         """
```
Comment (fgrunewald) [inline vermouth/gmx/itp_read.py:276]: I can't exactly remember why but for some inheritance reasons these were not static methods of the class. How badly do you wanna change this? I would advertise to come back to this as soon as we redo the parsers. I know it seems like almost an equivalent change but it had it's reasons.

Comment (fgrunewald) [reply vermouth/gmx/itp_read.py:276]: Same for the next three methods

Comment (pckroon) [reply vermouth/gmx/itp_read.py:276]: I don't think the staticmethod has any influence on the inheritance behaviour, but I'll change it.

**Pull request merged**